src/main/java/gregtech/common/blocks/BlockLampBorderless.java

```diff
@@ -7,18 +7,21 @@
 import net.minecraft.item.EnumDyeColor;
 import net.minecraft.util.BlockRenderLayer;
 
+import javax.annotation.Nonnull;
+
 public class BlockLampBorderless extends BlockLamp {
 
     public BlockLampBorderless(EnumDyeColor color) {
         super(color);
     }
 
     @Override
-    public boolean canRenderInLayer(IBlockState state, BlockRenderLayer layer) {
+    public boolean canRenderInLayer(@Nonnull IBlockState state, @Nonnull BlockRenderLayer layer) {
         return isLightActive(state) && state.getValue(BLOOM) && !Shaders.isOptiFineShaderPackLoaded() ?
-                layer == BloomEffectUtil.BLOOM : layer == BlockRenderLayer.SOLID;
+                layer == BloomEffectUtil.getBloomLayer() : layer == BlockRenderLayer.SOLID;
```

Comment:
Should this be `getEffectiveBloomLayer()`?

Reply:
The logic is correct, `isOptiFineShaderPackLoaded` check is already performed so the raw bloom instance can be used here. Technically.

Using new util method is better though, so I'll do that.